### PR TITLE
Add and Remove Labels From Multiple People

### PR DESCRIPTION
**Pull request by dcarello**
This program takes in names and a label that will be either applied or removed from all of the names depending on what the user chose

### CODE REVIEW

File: add_or_remove_labels.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        #initializes arguement parser
+        arg_parser = self.__initialize_argument_parser()
+        self.args = arg_parser.parse_args(cli_args)
+        self.api_url = "https://api2.rhombussystems.com"
+        #starts one api session that is used by all requests
+        self.api_sess = requests.session()
+        self.api_sess.headers = {
+            "x-auth-scheme": "api-token",
+            "x-auth-apikey": self.args.APIkey}
+        today = datetime.now().replace(microsecond=0, second=0, minute=0)
```
Comment (rhombusdev) [inline add_or_remove_labels.py]: don't need these time variables

File: add_or_remove_labels.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+    #formats the names to then put them in a list instead of one long string
+    def names(self):
+        self.args.names = self.args.names.replace(", ", ",")
+        self.names_list = self.args.names.split(",")
+
+    def execute(self):
+        self.names()
+        if self.args.choice == 'add':
+            for self.event, item in enumerate(self.names_list):
```
Comment (rhombusdev) [inline add_or_remove_labels.py]: This is awkward to read and understand.  Would be better if you just looped through the `self.names_list` list and passed the item to the `add_label` or `remove_label` functions.

**Review by rhombusdev — COMMENTED** [review]
Just some minor cleanup for readability.

**Comment by dcarello** [comment]
Cleanup complete

**Pull request merged**